### PR TITLE
Update link to the directory repository

### PR DESCRIPTION
**Pull request by rnestler**
To be merged as soon as the move from https://github.com/fixme-lausanne/OpenSpaceDirectory/issues/87 is done.

### CODE REVIEW

File: content/pages/directory.md [hunk 1]
```diff
@@ -4,9 +4,9 @@ Summary: Information about the SpaceAPI directory
 SortOrder: 40
 
 The directory hosts a list of hackerspaces which implement the SpaceAPI. It is
-currently maintained by the [FIXME Hackerspace](https://fixme.ch/).
+currently hosted by the [FIXME Hackerspace](https://fixme.ch/).
 
  * [Spaceapi directory](https://spaceapi.fixme.ch/directory.json)
  * Add your space to the directory by creating a pull request on
-   [Github](https://github.com/fixme-lausanne/OpenSpaceDirectory)
+   [Github](https://github.com/SpaceApi/OpenSpaceDirectory)
```
Comment (rnestler) [inline content/pages/directory.md]: Will we rename the repository (from OpenSpaceDirectory)? If yes what would be a good new name? @dbrgn ?

Comment (dbrgn) [reply content/pages/directory.md]: It's here: https://github.com/SpaceApi/directory/

Comment (rnestler) [reply content/pages/directory.md]: Done.

File: content/pages/directory.md [hunk 2]
```diff
@@ -4,9 +4,9 @@ Summary: Information about the SpaceAPI directory
 SortOrder: 40
 
 The directory hosts a list of hackerspaces which implement the SpaceAPI. It is
-currently maintained by the [FIXME Hackerspace](https://fixme.ch/).
+currently hosted by the [FIXME Hackerspace](https://fixme.ch/).
```
Comment (dbrgn) [inline content/pages/directory.md]: Is this still true? We now have https://directory.spaceapi.io/. Probably worth updating too.

Comment (rnestler) [reply content/pages/directory.md]: I compared the two endpoints and ours is still a bit buggy, see https://github.com/SpaceApi/website/issues/35.

Comment (dbrgn) [reply content/pages/directory.md]: Good point :slightly_smiling_face:

**Pull request merged**